blog/2024-08-14-conda-ecosystem-explained.mdx

```diff
@@ -0,0 +1,63 @@
+
+And yes, conda is free. Always has been, always will be. This commitment to open-source has led to the development of several compatible alternatives within the conda ecosystem, like [Mamba](https://mamba.readthedocs.io/) — a faster reimplementation in C++ \- and [Pixi](https://pixi.sh/), a newer, Rust-based package management tool.
+
+All of which use the same conda standards and formats, that are collaboratively improved through the [Conda Enhancement Proposal (CEP)](https://github.com/conda/ceps/blob/main/cep-1.md) process.
+
+## Conda Package Repository and Channels
+
+When it comes to managing packages with conda or conda-compatible tools, understanding where these packages come from is key. Anaconda.org serves as a central hub for conda packages, organized into various channels. While anyone can create their own channel on anaconda.org to host conda packages, we'll focus on the largest and most significant channels within the conda ecosystem.
+
+### `conda-forge`
+
+[conda-forge](https://conda-forge.org) is one of the largest and impactful channels in the conda ecosystem. Entirely community-driven, this channel hosts over 25,000 packages, making it a go-to resource for developers across the globe. Conda-forge is open-source, free to use, and maintained by a group of dedicated volunteers. The governance model ensures that the channel remains reliable and accessible to everyone.
+
+### Bioconda
+
+If your work involves computational biology, [Bioconda](https://bioconda.github.io) is the channel you’ll want to explore. Unlike conda-forge, which covers a wide variety of packages, Bioconda is focused on bioinformatics. It offers over 8,000 packages that cater to subdomains like genomics, proteomics, structural biology, and biomass analysis. Like conda-forge, bioconda is open-source and free to use, maintained by a community of volunteers.
+
+### `Defaults`
+
+The `defaults` channel is maintained by Anaconda, Inc. It's important to note that this channel comes with [Terms of Service](https://legal.anaconda.com/policies/en/). While largely free for individual users, students, and small companies, there are scenarios where a license might be required.
+
+## Getting Started with conda tools
+
+There are a few different ways to install and get started with conda:
+
+1. **Miniconda**: an installer developed by Anaconda, Inc. that installs conda and its dependencies. It is preconfigured to pull packages from the \`defaults\` channel, which has its own Terms of Service (as mentioned in conda channels section). You can always reconfigure conda after installation to use community channels like conda-forge or bioconda \[LINK to docs on how to do it\].
+2. **Miniforge**: developed by the conda-forge community, this installer comes with both conda and Mamba, and by default, it pulls packages from the conda-forge channel. It's a great option if you prefer a community-driven setup right from the start.
```

Comment:
```suggestion
2. **Miniforge**: developed by the conda-forge community, this installer comes with both `conda` and `mamba`, and by default, it pulls packages from the `conda-forge` channel. It's a great option if you prefer a community-driven setup right from the start.
```

Reply:
Maybe, to avoid the default (pre-selected option)  vs defaults (channel) we can phrase this like:

```
this installer comes with both `conda` and `mamba` and it pulls packages from the `conda-forge` channel instead of `defaults`. It's a great option if you prefer a community-driven setup right from the start.
```